next.config.mjs

```diff
@@ -5,7 +5,12 @@ const nextConfig = {
     appDir: true,
   },
   async rewrites() {
-    const ret = [];
+    const ret = [
+      {
+        source: "/api/proxy/:path*",
+        destination: "https://api.openai.com/:path*",
```

Comment:
Is this open proxy necessary for the feature?

Reply:
I need this proxy to use other chatgpt-based tools.

Reply:
@Yidadaa An open proxy could be abused by bad actors. Please consider securing this endpoint for deployments with `CODE` configured.

Reply:
I am trying to make it, this is a trade-off solution.